src/main/java/com/exactpro/th2/hand/builders/mstore/ProtobufMessageStoreBuilder.java

```diff
@@ -39,22 +38,21 @@
 
 import static com.exactpro.th2.hand.utils.Utils.getTimestamp;
 
-public final class DefaultMessageStoreBuilder implements MessageStoreBuilder<RawMessage> {
-	private static final Logger logger = LoggerFactory.getLogger(DefaultMessageStoreBuilder.class);
+public final class ProtobufMessageStoreBuilder implements MessageStoreBuilder<RawMessage> {
+	private static final Logger logger = LoggerFactory.getLogger(ProtobufMessageStoreBuilder.class);
 
-	private final ObjectMapper mapper = new ObjectMapper();
 	private final AtomicLong seqNum;
 	private final CommonFactory factory;
 
-	public DefaultMessageStoreBuilder(CommonFactory factory, AtomicLong seqNum) {
+	public ProtobufMessageStoreBuilder(CommonFactory factory, AtomicLong seqNum) {
 		this.factory = factory;
 		this.seqNum = seqNum;
 	}
 
 	@Override
 	public RawMessage buildMessage(Map<String, Object> fields, Direction direction, String sessionId, String sessionGroup) {
 		try {
-			byte[] bytes = mapper.writeValueAsBytes(fields);
+			byte[] bytes = CommonFactory.MAPPER.writeValueAsBytes(fields);
```

Comment:
Just want to note, that the mapper from CommonFactory is configured for CommonFactory and configuration deserialization. I am not sure  if we should use it anywhere else